File: uri/1st Page/cp with java/src/com/URI1008.java
import java.util.Scanner;
public class URI1008 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        int wh = sc.nextInt();
        double sh = sc.nextDouble();
        double slr = wh * sh;
        System.out.println("NUMBER = "+num);
        System.out.printf("SALARY = U$ %.2f\n",slr);
    }
}

File: uri/1st Page/cp with c/URI 1008.c
#include<stdio.h>
int main()
{
    int en, wh;
    float ar, SALARY;
    scanf("%d %d %f", &en, &wh, &ar);
    SALARY = wh*ar;
    printf("NUMBER = %d\n",en);
    printf("SALARY = U$ %.2f\n",SALARY);
    return 0;
}
